writer.go

```diff
@@ -4,9 +4,14 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
 	"strings"
+
+	isatty "github.com/mattn/go-isatty"
```

Comment:
You could avoid this dep with https://godoc.org/golang.org/x/crypto/ssh/terminal#IsTerminal, I think.

Reply:
Ok, let me try that.

Reply:
Ok, that avoided the dependency on `mattn/go-isatty`, but `dep` still needed to pull in a bunch of new dependencies.